tests/pod_builder.rs

```diff
@@ -0,0 +1,113 @@
+use k8s_openapi::api::core::v1::{Container, Pod, Volume, VolumeMount};
+use serde_json::json;
+use std::sync::Arc;
+
+pub struct PodLifetimeOwner {
+    pub pod: Pod,
+    _tempdirs: Vec<Arc<tempfile::TempDir>>, // only to keep the directories alive
+}
+
+pub struct WasmerciserContainerSpec {
+    pub name: &'static str,
```

Comment:
Any reason to not just make these `String`s?

Reply:
I don't want to clutter the calling code with `to_owned()`.  I want spec creation code focused on spec creation and not on optimising memory usage.  (If this were production code then yes I would totally make them `String`s though.)